data/human/weapons.txt

```diff
@@ -33,6 +33,36 @@ outfit "Energy Blaster"
 		"hull damage" 6.6
 	description "Although not the most accurate or damaging of weapons, the Energy Blaster is popular because it is small enough to be installed on even the tiniest of ships. One blaster is not enough to do appreciable damage to anything larger than a fighter, but a ship equipped with several of them becomes a force to be reckoned with."
 
+outfit "Twin Blaster"
+	category "Guns"
+	cost 40000
+	thumbnail "outfit/twin blaster"
+	"mass" 12
+	"outfit space" -12
+	"weapon capacity" -12
+	"gun ports" -1
+	weapon
+		sound "blaster"
+		"submunition" "Blaster Submunition"
+			"offset" 3 0
+		"submunition" "Blaster Submunition"
+			"offset" -3 0
```

Comment:
Six pixels apart looks very odd to me. At that distance it doesn't even look like the projectiles are coming out of the gun.
![image](https://user-images.githubusercontent.com/17688683/196828140-083c3f4c-d294-45a6-8bad-0c32bff20775.png)

I'd prefer something closer, like 3 pixels apart. At that distance it's about the thickness of the heavy laser turret, and looks more feasible from the point of the barrel.
![image](https://user-images.githubusercontent.com/17688683/196828164-0f1d55a1-7fb9-45b0-aa54-26e04ed8ae5e.png)
```suggestion
			"offset" 1.5 0
		"submunition" "Blaster Submunition"
			"offset" -1.5 0
```
Alternatively we can just double the fire rate as is done on the turrets.